docs/integrations/microk8s/k8s.mdx

```diff
@@ -0,0 +1,260 @@
+
+   ```bash
+   export NGROK_AUTHTOKEN=[YOUR-AUTHTOKEN]
+   export NGROK_API_KEY=[YOUR-API-KEY]
+   ```
+
+1. Install the ngrok Ingress Controller with Helm under a new `ngrok-ingress-controller` namespace, with the `useExperimentalGatewayApi=true` option.
+
+   ```bash
+   helm install ngrok-ingress-controller ngrok/kubernetes-ingress-controller \
+     --namespace ngrok-ingress-controller \
+     --create-namespace \
+     --set credentials.apiKey=$NGROK_API_KEY \
+     --set credentials.authtoken=$NGROK_AUTHTOKEN \
+     --set useExperimentalGatewayApi=true
+   ```
+
+1. Verify you have installed the ngrok Kubernetes Operator successfully via a single running pod.
+
+   ```bash
+   kubectl get pods --namespace ngrok-ingress-controller
+
+   NAME                                                              READY   STATUS    RESTARTS   AGE
+   ngrok-ingress-controller-kubernetes-ingress-controller-man2fg5p   1/1     Running   0          2m23s
+   ```
+
+## **Step 3**: Install an example app {#install-example-app}
+
+Now you need a domain and Kubernetes service to ingress traffic to!
+
+1. If you don't have an ngrok static domain, create one. Navigate to the [**Domains** section](https://dashboard.ngrok.com/cloud-edge/domains) of the ngrok dashboard and click **Create Domain** or **New Domain**. This static domain, which will look like `example.ngrok.app`, will be your `NGROK_DOMAIN` for the remainder of this guide.
+
+   Creating a subdomain on the ngrok network provides a public route to accept HTTP, HTTPS, and TLS traffic.
+
+1. Create a new Kubernetes manifest (`2048.yaml`) with the YAML below. This manifest defines the 2048 application service and deployment, then configures the ngrok Kubernetes Operator to connect the `game-2048` service to the ngrok edge via your `NGROK_DOMAIN`.
+
+   ```yaml showLineNumbers
+   apiVersion: v1
+   kind: Service
+   metadata:
+     name: game-2048
+     namespace: ngrok-ingress-controller
+   spec:
+     ports:
+       - name: http
+         port: 80
+         targetPort: 80
+     selector:
+       app: game-2048
+   ---
+   apiVersion: apps/v1
+   kind: Deployment
+   metadata:
+     name: game-2048
+     namespace: ngrok-ingress-controller
+   spec:
+     replicas: 1
+     selector:
+       matchLabels:
+         app: game-2048
+     template:
+       metadata:
+         labels:
+           app: game-2048
+       spec:
+         containers:
+           - name: backend
+             image: alexwhen/docker-2048
+             ports:
+               - name: http
+                 containerPort: 80
+   ```
+
+1. Apply the `2048.yaml` manifest to your MicroK8s cluster.
+
+   ```bash
+   kubectl apply -f 2048.yaml
+   ```
+
+   :::tip
+
+   **Note:** If you get an error when applying the manifest, double check that you've updated the `NGROK_DOMAIN` value
```

Comment:
I'm thinking this `Note` should go down below the `Gateway` configuration below?